tema-css-2/tema-2-positioning/style.css

```
@@ -0,0 +1,37 @@
+.parent {
+    background-color: rgb(187, 201, 184);
+    width: 350px;
+    height: 130px;
+    border: 2px dotted red;
+    border-radius: 6px;
+    text-align: center;
+    position: relative;
+}
+.child {
+    width: 130px;
+    height: 90px;
+    border-radius: 6px;
+    border: 1.5px dotted red; 
+}
+.child1 {
+    background-color: rgb(81, 141, 138);
+    top: 0;
+    left: 0;
+    position: absolute;
+}
+.child2 {
+    background-color: rgb(163, 145, 164);
+    top: 30%;
```

Aici, pentru ca al 2lea copil este lipit de partea de jos, mai indicat ar fi sa ii setezi proprietatea de bottom

```suggestion
    bottom: 0;
```